Frontend/src/components/TableEditors/Rentals/Rentals.svelte

```diff
@@ -24,6 +24,19 @@
     return isBeforeDay(millis, new Date().getTime());
   }
 
+  function cellBackgroundColorFunction(col, item) {
+    if (col.key == "customer_id" || col.key == "name") {
+      $customerDb.fetchById(item.customer_id).then(function (doc) {
```

Comment:
this does not work. the fetchById returns a promise, however, when I later return via doc.highlight (which correctly returns "red" if I have selected it), but that never gets updated. @daniel17903 how can I make the table update with this value?